src/Channel.cpp

```diff
@@ -214,3 +190,25 @@ void Channel::deactivateLimitMode(void) {
   std::cout << "[" << _nameWithPrefix << "] Limit mode desactivated"
             << std::endl;
 }
+
+
+/*============================================================================*/
+/*       Others                                                               */
+/*============================================================================*/
+
+void Channel::receiveMessageInTheChannel(int fd) {
+  if (_channelClients.find(fd) != _channelClients.end()) {
+    std::string message = _channelClients[fd]->shareMessage();
+    if (!message.empty()) {
+      std::cout << "Message received in channel " << _name << " from client "
+                << fd << ": " << message << std::endl;
+      clientPMap::iterator itBegin = _channelClients.begin();
+      clientPMap::iterator itEnd = _channelClients.end();
+      for (clientPMap::iterator it = itBegin; it != itEnd; ++it) {
+        if (it->first != fd) {
+          it->second->receiveMessage(message);
+        }
+      }
+    }
+  }
+}
```

Comment:
I didn't test this function because it's not used.

Reply:
oui elle avait ete faite au tout debut par melodie dans bidouille. je la laisse pour la reprendre